src/OSPSuite.Core/Domain/ValueOrigin.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public class ValueOrigin
```

Comment:
This is the new class. Fairly simple with the OriginType and description together